eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -61,9 +65,9 @@ jobs:
         name: azsdk-pool-mms-win-2019-general
         vmImage: MMS2019
       variables:
-        VcpkgDependencies: curl[winssl] libxml2
         VCPKG_DEFAULT_TRIPLET: 'x64-windows-static'
         Package.EnableSBOMSigning: true
+        VCPKG_CACHE_MODE: "read"
```

Comment:
here you say read

Reply:
yes, since this is public pipeline, we only allow read mode.
And in case you wonder, if anyone feels smart and change this to `readwrite` and run the pipeline; the pipeline will ask vcpkg to write built binaries, but VCPKG will fail to push the binaries because only the `internal` pipelines are instrumented with writting access.